arangod/Cache/TransactionManager.cpp

```diff
@@ -67,7 +68,7 @@ Transaction* TransactionManager::begin(bool readOnly) {
         state, newState, std::memory_order_acq_rel, std::memory_order_relaxed));
   }
   tx->term = newState.term;
-  return tx;
+  return tx.release();
```

Comment:
As it's leaving the scope, isn't the `unique_ptr` gonna release anyway?

Reply:
The unique_ptr would deallocate the object when going out of scope here.
But we want to return the underlying pointer. This is what `release()` does: we basically tell the unique_ptr to give up its ownership, because we want to manage the pointer ourselves and return it.